x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations.tsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiPageContent,
+  EuiTitle,
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiEmptyPrompt,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { FlashMessages } from '../../../../shared/flash_messages';
+import { KibanaLogic } from '../../../../shared/kibana';
+import { Loading } from '../../../../shared/loading';
+import { EuiButtonTo, EuiLinkTo } from '../../../../shared/react_router_helpers';
+import { ENGINE_CURATIONS_NEW_PATH, ENGINE_CURATION_PATH } from '../../../routes';
+import { generateEnginePath } from '../../engine';
+
+import { CURATIONS_OVERVIEW_TITLE, CREATE_NEW_CURATION_TITLE } from '../constants';
+import { CurationsLogic } from '../curations_logic';
+import { Curation } from '../types';
+
+export const Curations: React.FC = () => {
+  const { dataLoading, curations, meta } = useValues(CurationsLogic);
+  const { loadCurations } = useActions(CurationsLogic);
+
+  useEffect(() => {
+    loadCurations();
+  }, [meta.page.current]);
+
+  if (dataLoading && !curations.length) return <Loading />;
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>{CURATIONS_OVERVIEW_TITLE}</h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+        <EuiPageHeaderSection>
+          <EuiButtonTo to={generateEnginePath(ENGINE_CURATIONS_NEW_PATH)} fill>
+            {CREATE_NEW_CURATION_TITLE}
+          </EuiButtonTo>
+        </EuiPageHeaderSection>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <FlashMessages />
+        <CurationsTable />
+      </EuiPageContent>
+    </>
+  );
+};
+
+export const CurationsTable: React.FC = () => {
```

Comment:
I think this should go into it's own curations_table.tsx file

Reply:
This is totally optional though.

Reply:
I'm leaning towards leaving these in the same file for a few reasons:

- CurationsTable isn't used anywhere else but this one view
- The components are mainly separated out for easier readability and easier unit testing, not because they're conceptually separate
- They do have some intermingled logic (primarily `useEffect`/`loadCurations()` whenever pagination changes)

Hope that all makes sense!